tests/snapshot.rs

```diff
@@ -38,7 +38,7 @@ struct Report {
 }
 
 #[test]
-#[traced_test]
+// #[traced_test]
```

Comment:
Why?

Reply:
commented out all tests for debugging, forget to change it back.